lib/spack/spack/compilers/msvc.py

```diff
@@ -154,10 +154,12 @@ class Msvc(Compiler):
 
     #: Regex used to extract version from compiler's output
     version_regex = r"([1-9][0-9]*\.[0-9]*\.[0-9]*)"
-
+    suffixes = [""]
     # Initialize, deferring to base class but then adding the vcvarsallfile
```

Comment:
Does this comment belong here? It looks like it was originally meant to discuss the `__init__` method.

Reply:
This comment: `# Initialize, deferring to base class but then adding`? that was not something that was touched in this PR, but you're right, it was. I'll move it to the correct location right before the call to `super`.